tests/bcl-test/Makefile

```diff
@@ -0,0 +1,22 @@
+TOP = ../../..
```

Comment:
Same: this file does not exist in main anymore.

Reply:
Same, a mistake, will remove